client/benches/torii.rs

```diff
@@ -64,7 +64,7 @@ fn query_requests(criterion: &mut Criterion) {
     let create_domain = Register::domain(Domain::new(domain_id.clone()));
     let account_id = AccountId::new(domain_id.clone(), "account".parse().expect("Valid"));
     let (public_key, _) = KeyPair::generate().into();
-    let create_account = Register::account(Account::new(account_id.clone(), [public_key]));
+    let create_account = Register::account(Account::new(account_id.clone(), public_key));
```

Comment:
hm, this is actually, quite a good compromise

Reply:
Yes. I also took a look at some crates which provides things like `NonEmptyIterator` and `NonEmptyVec`, but it looks like having such dependencies doesn't worth it in this case